packages/core/src/configuration-non-customizable.ts

```diff
@@ -57,10 +57,41 @@ export const legend = {
 		horizontalSpace: 12,
 		verticalSpace: 24,
 		textYOffset: 8,
+		spaceAfter: 4,
 	},
 	checkbox: {
 		radius: 6.5,
-		spaceAfter: 4,
+	},
+	radius: {
+		config: [
```

Comment:
what does `config` do? What does it describe? could any of these fields be more detailed?

Reply:
it provides the config data for the icon. I changed it to `iconData` since it's an array.